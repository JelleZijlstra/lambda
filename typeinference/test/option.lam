data option a = None | Some a in
None
