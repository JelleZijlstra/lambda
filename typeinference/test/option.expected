%%stdout%%
Result: module
	let __result__ = None
end
%%stderr%%
